### ISSUE TITLE
Graph Visualizer: turning off node collapsing

### ISSUE THREAD

**Issue by yaroslavvb** [tensorflow/tensorboard#572]
TensorBoard collapses Relu-Relu_6 nodes together into one node which turns simple neural net graphs into a jumple

Example pbtxt [here](https://github.com/yaroslavvb/stuff/blob/master/resnet_8_simple.pbtxt) 
from resnet-8 from https://github.com/tensorflow/models

Current TensorBoard

![screenshot 2017-09-25 11 35 53](https://user-images.githubusercontent.com/23068/30824844-bf81f922-a1e5-11e7-9cf4-5995ba19e5a4.png)

Here's after I hack it to stop collapsing Relu nodes. Note that softmax layer is now at the top as expected, instead of middle of the graph as before

![screenshot 2017-09-25 12 16 59](https://user-images.githubusercontent.com/23068/30826436-775c2f86-a1eb-11e7-8532-1d895014f17d.png)

**Comment by chihuahua** [comment]
Hmm, one quick thing you can do is to select the series node and then click "Ungroup this series of nodes":

![iqrnm9qxdfb](https://user-images.githubusercontent.com/4221553/31051019-454390f4-a612-11e7-9076-16c36c24c6d7.png)

**Comment by yaroslavvb** [comment]
Another example pbtxt, it's resnet 14 cifar, you have to spend a few minutes uncollapsing nodes before it becomes visualizable
[cifar_14.txt](https://github.com/tensorflow/tensorboard/files/1444704/cifar_14.txt)

**Comment by bileschi** [comment]
@davidsoergel is there a master issue for graph management?
